### PR TITLE
[AI] Parler Text to Speech

### PR DESCRIPTION
**Pull request by pschroedl**
This pull request introduces an implementation of the Text to Speech pipeline within the AI worker.

This implements the basic functionality of the Parler Text-to-Speech model (https://huggingface.co/parler-tts/parler-tts-large-v1) by supplying text_input and description fields - the description is used to ‘steer’ the qualities of the voice synthesized by the model.

This PR follows the method of adding pipelines dependencies without breaking existing implementations present in ai-runner by using an additional dockerfile definition. Example usage for local development and testing:

```
cd ai-worker/runner
docker build -t livepeer/ai-runner:base .
docker build -f docker/Dockerfile.text_to_speech -t livepeer/ai-runner:text-to-speech .

docker run --name text-to-speech -e DEEPCACHE=True -e PIPELINE=text-to-speech -e MODEL_ID=parler-tts/parler-tts-large-v1 -e HUGGINGFACE_TOKEN={token} --gpus all -p 47906:8000 -v ~/.lpData/models:/models livepeer/ai-runner:text-to-speech

```

### CODE REVIEW

**Comment by pschroedl** [comment]
ai-worker bindings, container image, and go-livepeer gateway + orch have been rebuilt and can be validated/tested via : [https://colab.research.google.com/drive/12UN65fV6q44IyMRL2tyKFal8OMCyyRMN](https://colab.research.google.com/drive/12UN65fV6q44IyMRL2tyKFal8OMCyyRMN)

**Review by rickstaa — APPROVED** [review]
@pschroedl approved. Let's rebase the go-livepeer version and check if we can get rid of the base64 encoding between the runner and worker 👍🏻.

File: runner/docker/Dockerfile.text_to_speech [hunk 1]
```diff
@@ -0,0 +1,13 @@
+ARG BASE_IMAGE=livepeer/ai-runner:base
+FROM ${BASE_IMAGE}
+
+RUN pip install --no-cache-dir \
+    transformers==4.43.0 \
+    peft \
+    deepcache \
+    soundfile \
+    g2p-en \
+    # flash_attn \  # TODO (pschroedl): requires cuda-toolkit
```
[the remark this answers is not in the document]
Comment (rickstaa) [reply runner/docker/Dockerfile.text_to_speech]: Solved.

File: runner/docker/Dockerfile.text_to_speech [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    transformers==4.43.3 \
+    peft \
+    deepcache \
+    soundfile \
+    g2p-en \
+    flash_attn \
+    git+https://github.com/huggingface/parler-tts.git@5d0aca9753ab74ded179732f5bd797f7a8c6f8ee
+
+# Copy the app directory into the container
+COPY app /app
```
Comment (rickstaa) [inline runner/docker/Dockerfile.text_to_speech:19]: @pschroedl this is already done in the base container right (see https://discord.com/channels/@me/1219448753635725443/1301676937114025995)? If this is not needed due to some errors lets remove it and then this pull request is good to be merged 🚀.

**Review by rickstaa — APPROVED** [review]
@pschroedl please merge after last comment is resolved 👍🏻.

File: runner/docker/Dockerfile.text_to_speech (hunk 2 above)
Comment (rickstaa) [reply runner/docker/Dockerfile.text_to_speech:19]: @pschroedl stated that this was to prevent having to rebuild the base container during debugging.

**Pull request merged**